src/app/shell/Destiny.tsx

```diff
@@ -20,42 +20,38 @@ import ItemPopupContainer from '../item-popup/ItemPopupContainer';
 import ErrorPanel from './ErrorPanel';
 
 // TODO: Could be slightly better to group these a bit, but for now we break them each into a separate chunk.
-const Inventory = React.lazy(
-  () => import(/* webpackChunkName: "inventory" */ 'app/inventory/Inventory')
```

Comment:
You have something going on with your linting. If I open these and save it reformats everything. Can you check that you have eslint and prettier set up correctly and pull master resave these files for me. 

I am assuming you are using vscode with the eslint and prettier plugins btw, if you aren't you might need to do something to set that stuff up.

Reply:
Found the dodgy setting, fixed.